source/core/index.ts

```diff
@@ -176,6 +176,7 @@ export default class Request extends Duplex implements RequestEvents<Request> {
 	private _triggerRead: boolean;
 	declare private _jobs: Array<() => void>;
 	private _cancelTimeouts: () => void;
+	private readonly _removeListeners: () => void;
```

Comment:
```suggestion
	private _removeListeners: () => void;
```

Reply:
`xo` does not like this suggestion without reassigning `_removeListeners` somewhere. Should it be reset to `noop` somewhere?